This patch forces the addition of local changes (e.g., removal of `/bin/pwd'
from `configure') to the tarball.  Use `--diff-filter' to filter out deleted
files.

With Git 1.7.6, the `git diff' command below isn't enough to show added but
untracked files (it worked with 1.7.1), hence the additional `git status'.

diff --git a/Makefile b/Makefile
index 61cfa43..a6e7cee 100644
--- a/Makefile
+++ b/Makefile
@@ -363,6 +363,11 @@ dist-prepare: $(files-for-dist)
 
 %.tar: FORCE
 	git archive --prefix=$*/ $* > $@.new
+	tar -rv --owner=0 --group=0		\
+	  --transform='s|^\(.*\)$$|$*/\1|g'	\
+	  -f "$@.new"				\
+	  `git diff --name-only --diff-filter=ACMT` \
+	  `git status --porcelain | grep '^\?' | cut -d ' ' -f 2-`
 	mv -f $@.new $@
 
 # Do `make dist dist-version=X.Y.Z' to make tar files of an older version.
